packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js

```diff
@@ -148,7 +148,11 @@ const runGenerator = () => {
     // Shelljs can't run interactive programs, so we have to switch to child_process.
     // See https://github.com/shelljs/shelljs/wiki/FAQ#running-interactive-programs-with-exec
 
-    const flags = semver.satisfies(process.version, '>=15') ? '-y' : ''
+    const extension = process.platform === 'win32' ? '.cmd' : ''
+    const npm = `npm${extension}`
+    const foundNpm = cp.spawnSync(npm, ['-v']).stdout.toString().trim()
+    const flags = semver.satisfies(foundNpm, '>=7') ? '-y' : ''
```

Comment:
The `-y` flag was added to `npx` in npm 7 ([link](https://docs.npmjs.com/cli/v7/commands/npx#compatibility-with-older-npx-versions)). To find the current `npm` version, I'm following the `win32` logic that's in `check-version.js` [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/scripts/check-version.js#L10-L13).